### ISSUE TITLE
Grid Column / Row Span for IE11

### ISSUE THREAD

**Issue by gisu** [postcss/autoprefixer#1084]
The autoprefixer has slight problems with changing the column widths. 

Example a grid with three columns. In the first breakpoint, an element has 2 columns. 

```scss
grid-gap: 10px 20px;
grid-template:
  "log met qu2" 20px
  "log lin lin" auto
  "log nav nav" auto /
  200px 1fr 110px;
```

In the next breakpoint, the element `nav` should now work on a column.

```scss
grid-gap: 10px 50px;
grid-template:
  "log met soc" 20px
  "log lin lin" auto
  "log nav qu2" auto /
  290px 1fr 110px;
```

The problem here is that in the second grid I have `-ms-grid-column-span: 3` still from the previous breakpoint. Which means that the elements now overlap. I have now helped me manually add `-ms-grid-column-span: 1` at the breakpoint.

To automate the whole thing, the autoprefixer should always note `-ms-grid-column-span`, even if the box is only one column wide to overwrite old values in CSS. This also applies to the `-ms-grid-row-span`.

**Comment by ai** [comment]
Can you show full input and output?

**Comment by gisu** [comment]
I have reduced the input and output by the most important things

```scss
// [root]__inner
&__inner {
  display: grid;

  @include break('m') {
    grid-gap: 10px 20px;
    grid-template:
      "log nav nav" auto /
        200px 1fr 110px;
  }

  @include break('l') {
    grid-gap: 10px 50px;
    grid-template:
      "log nav qu2" auto /
      290px 1fr 110px;
  }
}

// [root]__log
&__log {
  grid-area: log;
}

// [root]__nav
.c-mainnav {
  display: none;
  grid-area: nav;
  @include break(apin(2, mq)) {
    display: block;
  }
}
```

Output
```css
.c-pageHeader__inner {
  display: -ms-grid;
  display: grid;
}

@media screen and (min-width: 768px) {
  .c-pageHeader__inner {
    grid-gap: 10px 20px;
    -ms-grid-rows: auto;
    -ms-grid-columns: 200px 20px 1fr 20px 110px;
    grid-template: "log nav nav" auto / 200px 1fr 110px;
  }
}

@media screen and (min-width: 1340px) {
  .c-pageHeader__inner {
    grid-gap: 10px 50px;
    -ms-grid-rows: auto;
    -ms-grid-columns: 290px 50px 1fr 50px 110px;
    grid-template: "log nav qu2" auto / 290px 1fr 110px;
  }
}

.c-pageHeader__log {
  grid-area: log;
}

.c-pageHeader .c-mainnav {
  display: none;
  grid-area: nav;
}

@media screen and (min-width: 768px) {
  .c-pageHeader__log {
    -ms-grid-row: 1;
    -ms-grid-column: 1;
  }
  .c-pageHeader .c-mainnav {
    -ms-grid-row: 1;
    -ms-grid-column: 3;
    -ms-grid-column-span: 3;
  }
}

@media screen and (min-width: 768px) {
  .c-pageHeader .c-mainnav {
    display: block;
  }
}

.c-pageHeader .c-quicklinksDesktop {
  display: none;
  grid-area: qu2;
}

@media screen and (min-width: 1340px) {
  .c-pageHeader__log {
    -ms-grid-row: 1;
    -ms-grid-column: 1;
  }
  .c-pageHeader .c-mainnav {
    -ms-grid-row: 1;
    -ms-grid-column: 3;
  }
  .c-pageHeader .c-quicklinksDesktop {
    -ms-grid-row: 1;
    -ms-grid-column: 5;
  }
}
```

**Comment by ai** [comment]
Fix released in 9.1
